### ISSUE TITLE
Compile Step Request: Insert Frontmatter

### ISSUE THREAD

**Issue by kevboh** [kevboh/longform#41]
(inspired by #37)

A compile step that takes a single option, which is a path to another note in the vault. Ideally the step UI will be updated in such a way that the option can autocomplete (perhaps by introducing a new option type?).

When the step runs, it takes the frontmatter in the note at its given option path and merges that frontmatter into either all input scenes or the manuscript.

I thought about refactoring step options to add, like, ad-hoc options that are each frontmatter entries, but that sounds like a ton of work _and_ I think offloading frontmatter composition to a note is safer and easier all around.

**Comment by wskort** [comment]
Yes please! I've tried tricking the system by having comments disrupt the frontmatter on the top chapter, but it didn't quite work out as planned.
